src/main/java/io/github/thebusybiscuit/sensibletoolbox/items/MultiBuilder.java

```diff
@@ -60,14 +60,6 @@ public MultiBuilder() {
         charge = 0;
     }
 
-    public MultiBuilder(ConfigurationSection conf) {
-        super(conf);
-        mode = Mode.valueOf(conf.getString("mode"));
-        charge = conf.getDouble("charge");
-        String s = conf.getString("material");
-        material = s.isEmpty() ? null : Material.matchMaterial(s);
-    }
-
```

Comment:
Pretty sure that constructor is required...

Reply:
IntelliJ marked it in grey and the project compiles fine after removing it, it's the only machine for which it's greyed out though. I guess I can add it back

Reply:
I am not 100% sure but I think that constructor is invoked via reflection, so it may not show as used even if it is.

Reply:
Makes sense